guides/05_tabular_data_science_and_plots/creating_a_realtime_dashboard_from_google_sheets.md

```diff
@@ -0,0 +1,147 @@
+# Creating a Real-Time Dashboard from Google Sheets
+
+Tags: TABULAR, DASHBOARD, PLOTS 
+
+[Google Sheets](https://www.google.com/sheets/about/) are an easy way to store tabular data in the form of spreadsheets. With Gradio and pandas, it's easy to read data from public or private Google Sheets and then display the data or plot it. In this blog post, we'll build a small *real-time* dashboard, one that updates when the data in the Google Sheets updates. 
+
+Building the dashboard itself with just be 9 lines of Python code using Gradio, and our final dashboard will look like this:
+
+<gradio-app space="gradio/line-plot"></gradio-app>
+
+**Prerequisites**: This Guide uses [Gradio Blocks](../01_getting_started/01_quickstart.md#blocks-more-flexibility-and-control), so make your are familiar with the Blocks class. 
```

Comment:
typo 
> so make your are familiar

so make sure* you're* familiar

Reply:
This Gradio Blocks link is incorrect, not sure where you got it from, but it should be: 

`[Gradio Blocks](../quickstart/#blocks-more-flexibility-and-control)`

Reply:
Oh okay yeah